Unity/Assets/MyScripts/ShowRewardedVideoScript.cs

```diff
@@ -1,168 +1,79 @@
+    {
+        Debug.Log("unity-script: I got RewardedVideoOnAdOpenedEvent With AdInfo " + adInfo);
+    }
 
-    void RewardedVideoAdLoadedDemandOnlyEvent(string instanceId){
-        ShowText.GetComponent<UnityEngine.UI.Text>().color = UnityEngine.Color.blue;
-        Debug.Log("unity-script: I got RewardedVideoAdLoadedDemandOnlyEvent for instance: " + instanceId);
+    void RewardedVideoOnAdClosedEvent(IronSourceAdInfo adInfo)
+    {
+        Debug.Log("unity-script: I got RewardedVideoOnAdClosedEvent With AdInfo " + adInfo);
     }
 
-    void RewardedVideoAdLoadFailedDemandOnlyEvent(string instanceId, IronSourceError error)
+    void RewardedVideoOnAdAvailable(IronSourceAdInfo adInfo)
     {
-        ShowText.GetComponent<UnityEngine.UI.Text>().color = UnityEngine.Color.red;
-        Debug.Log("unity-script: I got RewardedVideoAdLoadFailedDemandOnlyEvent for instance: " + instanceId + ", code :  " + error.getCode() + ", description : " + error.getDescription());
+        Debug.Log("unity-script: I got RewardedVideoOnAdAvailable With AdInfo " + adInfo);
     }
 
-    void RewardedVideoAdOpenedDemandOnlyEvent (string instanceId)
-	{
-		Debug.Log ("unity-script: I got RewardedVideoAdOpenedDemandOnlyEvent for instance: " + instanceId);
-	}
+    void RewardedVideoOnAdUnavailable()
+    {
+        Debug.Log("unity-script: I got RewardedVideoOnAdUnavailable");
+    }
 
-	void RewardedVideoAdRewardedDemandOnlyEvent (string instanceId)
-	{
-        Debug.Log("unity-script: I got RewardedVideoAdRewardedDemandOnlyEvent for instance: " + instanceId);
-	}
-	
-	void RewardedVideoAdClosedDemandOnlyEvent (string instanceId)
-	{
-		Debug.Log ("unity-script: I got RewardedVideoAdClosedDemandOnlyEvent for instance: " + instanceId);
-	}
-	
-	void RewardedVideoAdShowFailedDemandOnlyEvent (string instanceId, IronSourceError error)
-	{
-		Debug.Log ("unity-script: I got RewardedVideoAdShowFailedDemandOnlyEvent for instance: " + instanceId + ", code :  " + error.getCode () + ", description : " + error.getDescription ());
-	}
+    void RewardedVideoOnAdShowFailedEvent(IronSourceError ironSourceError, IronSourceAdInfo adInfo)
+    {
+        Debug.Log(
+            "unity-script: I got RewardedVideoAdOpenedEvent With Error" + ironSourceError + "And AdInfo " + adInfo);
+    }
 
-	void RewardedVideoAdClickedDemandOnlyEvent (string instanceId)
-	{
-        Debug.Log("unity-script: I got RewardedVideoAdClickedDemandOnlyEvent for instance: " + instanceId);
-	}
+    void RewardedVideoOnAdRewardedEvent(IronSourcePlacement ironSourcePlacement, IronSourceAdInfo adInfo)
+    {
+        Debug.Log("unity-script: I got RewardedVideoOnAdRewardedEvent With Placement" + ironSourcePlacement +
+                  "And AdInfo " + adInfo);
+    }
 
+    void RewardedVideoOnAdClickedEvent(IronSourcePlacement ironSourcePlacement, IronSourceAdInfo adInfo)
+    {
+        Debug.Log("unity-script: I got RewardedVideoOnAdClickedEvent With Placement" + ironSourcePlacement +
```

Comment:
@virginiecrequit You are missing spaces in the log

Reply:
```suggestion
        Debug.Log("unity-script: I got RewardedVideoOnAdClickedEvent With Placement " + ironSourcePlacement +
```

@alon357 Just to be sure, you mean like that? (So the space before the + or do you mean somewhere else too?)

Reply:
yes